3c6105073b70:
  acc_bias:
  - -0.11577714236427435
  - 0.019077577097797102
  - 0.868452140461766
  acc_scaler:
  - 0.10233696451220485
  - 0.0
  - 0.0
  - -0.0001418716309799564
  - 0.1023038272432077
  - 0.0
  - -0.00031281486621949963
  - 1.7415355974072063e-05
  - 0.10237798106404955
  gyro_bias:
  - 0.00432557532951773
  - 0.05439953844820461
  - 0.025777293656933778
6ca3b8:
  acc_bias:
  - 0.023734849828421174
  - -0.10868200267707888
  - 0.05794119279091827
  acc_scaler:
  - 0.10227108091970935
  - 0.0
  - 0.0
  - -0.0008043120238009737
  - 0.1020420165088371
  - 0.0
  - 0.0002635114626995536
  - 0.0010257840422153196
  - 0.10188082750768349
  gyro_bias:
  - -0.07208091620370136
  - -0.03692190216940321
  - 0.0010181106741233925
a00124:
  acc_bias:
  - 0.05081096513254256
  - -0.03473908737487149
  - 0.0949555517290153
  acc_scaler:
  - 0.10206645190172477
  - 0.0
  - 0.0
  - 0.00015988960270332743
  - 0.10200307952005873
  - 0.0
  - -0.00023671295339217028
  - 2.751977187034646e-05
  - 0.10177633059952701
  gyro_bias:
  - -0.014952129513121247
  - -0.0769557232395216
  - 0.007837516683091104
a00ca0:
  acc_bias:
  - 0.009907224791613215
  - -0.20402386503551145
  - 0.10177348958887378
  acc_scaler:
  - 0.10209718753606394
  - 0.0
  - 0.0
  - -0.00028898072133881896
  - 0.10187429302638709
  - 0.0
  - 5.638429400682228e-05
  - 0.0005154914338393937
  - 0.10198066008489107
  gyro_bias:
  - -0.018308488435465987
  - -0.037052285470088375
  - 0.0014946218665752103
a00fc8:
  acc_bias:
  - -0.014467202309686745
  - -0.20486113869858544
  - 0.04694124625362308
  acc_scaler:
  - 0.10192117241369228
  - 0.0
  - 0.0
  - 0.000248833831891536
  - 0.10186215019514129
  - 0.0
  - -0.00044851241588442155
  - -0.00012341423102235486
  - 0.10220133238787288
  gyro_bias:
  - -0.07194366328270478
  - -0.1543253976642152
  - -0.01205488646386989
a016d8:
  acc_bias:
  - 0.01785969429675878
  - -0.061266791622973826
  - 0.004889309703384947
  acc_scaler:
  - 0.10209551803242377
  - 0.0
  - 0.0
  - -0.0003535461755181247
  - 0.10185926778942685
  - 0.0
  - -9.918650656540727e-05
  - -0.00010703208191911986
  - 0.10206143631478237
  gyro_bias:
  - -0.0932100763094518
  - -0.059918793812909536
  - 0.017796894361437776
a0201c:
  acc_bias:
  - -0.03571501876073564
  - -0.07700040596455224
  - 0.02283119956422058
  acc_scaler:
  - 0.10213822491933296
  - 0.0
  - 0.0
  - -7.538408085677255e-05
  - 0.10184829986933704
  - 0.0
  - -8.867145458236054e-07
  - 0.00045334961950901047
  - 0.10198622785901258
  gyro_bias:
  - -0.019589689043572434
  - -0.06159376582506947
  - 0.004785740178309145
a4cf126d6f10:
  acc_bias:
  - -0.19113856544083052
  - -0.02541012751937886
  - 0.7475218940789635
  acc_scaler:
  - 0.10223474117898872
  - 0.0
  - 0.0
  - 2.1778105925628895e-05
  - 0.10218752445917302
  - 0.0
  - -5.140959546035421e-05
  - -2.032012005491056e-05
  - 0.10222838452079894
  gyro_bias:
  - -0.41976618002777705
  - 0.028406101015917538
  - 0.3134756115783631
